app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
     <activity
-      android:name=".home.continueplaying.ContinuePlayingActivity"
+      android:name=".home.recentlyplayed.RecentlyPlayedActivity"
```

Comment:
Alphabetical ordering of this `activity` will change, it should come after `.home.HomeActivity`

Reply:
Done